ui/app/AppLayouts/Wallet/components/TokenSettingsModalContent.qml

```diff
@@ -79,12 +79,14 @@ Item {
                         //% "Token details"
                         text: qsTrId("token-details")
                         onTriggered: {
-                            console.log("TODO")
+                            addCustomTokenModal.open()
+                            addCustomTokenModal.setData(address, name, symbol, decimals)
```

Comment:
You can set the data first, that way, there is no chance of the input "flashing"